<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>链助手</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>871</width>
     <height>721</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText</set>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>代币空投</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_config">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText</set>
     </property>
     <property name="title">
      <string>1、配置文件</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_address">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText</set>
      </property>
      <property name="text">
       <string>0xd3c0b6Aa1538d639912789be705F18b5Fd89fcE6</string>
      </property>
      <property name="placeholderText">
       <string>用户地址</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_chainId">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText</set>
      </property>
      <property name="text">
       <string>1320</string>
      </property>
      <property name="placeholderText">
       <string>链ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_private">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="text">
       <string>e714d873c046aa25df761d7f0f263caa2d4594778c8174e075bfeb26a9b968f1</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>用户私钥</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_gas">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText</set>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="placeholderText">
       <string>gas价格</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_config">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>确认设置</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_url">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText</set>
      </property>
      <property name="text">
       <string>http://http-testnet.aitd.io</string>
      </property>
      <property name="placeholderText">
       <string>url</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_contract">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText</set>
      </property>
      <property name="text">
       <string>0x1657a0E85F38640bc44dADAcfE84780301D4Ae2E</string>
      </property>
      <property name="placeholderText">
       <string>空投代币合约地址</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_approve">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>351</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>3、授权代币</string>
     </property>
     <widget class="QPushButton" name="pushButton_approve">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>点击授权代币</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_sendToken">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>300</y>
       <width>351</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>4、发送代币</string>
     </property>
     <widget class="QPushButton" name="pushButton_send">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>点击发送代币</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_address">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>341</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>2、上传用户地址</string>
     </property>
     <widget class="QTextEdit" name="textEdit_address">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>321</width>
        <height>191</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>输入地址和数量，以逗号分隔</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_csv">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>点击上传csv文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_csvOk">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>操盘助手</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_config2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>801</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>1、配置账号</string>
     </property>
     <widget class="QTextEdit" name="textEdit_privatekey2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>561</width>
        <height>81</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>每个私钥一行，程序不会收集您的私钥</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addPrivateKey2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;s &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>添加私钥</string>
      </property>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget_model2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>801</width>
       <height>341</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>刷量模式</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_tokenAmountLeft_A_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>200</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_tokenAmountRight2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>300</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_timeIntervalLeft2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>500000</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>代币数量区间</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>时间间隔</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Gas price</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Gas limit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>滑点</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>拉盘</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>砸盘</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>震荡模式</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_configPool2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>781</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>2、配置池子信息</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_lpAddress2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>LP地址</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_route2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>路由地址</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pool2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>链URL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_url2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>链ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_chainId2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_getAccountInfo2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>140</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>获取账号信息</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>verticalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_addPrivateKey2</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_privatekey2</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
